tests/utils/contract_utils.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from typing import Callable, Any, Optional
+
+from blockchain.contracts.base_interface import ContractInterface
+
+
+def check_contract(
+    contract: ContractInterface,
+    functions_spec: list[tuple[str, Optional[tuple], Callable[[Any], None]]],
+    caplog,
```

Comment:
What is `caplog` here?

Reply:
This is pytest feature that captures logs in tests